holidayextras/brands
# Moving ui-brand-heha to brands

#### What does this PR do? (please provide any background)
Moves the latest [ui-brand-heha](https://github.com/holidayextras/ui-brand-heha) to this repository to match the uxui roadmap.

#### What tests does this PR have?
None

#### How can this be tested?
- Pull latest `master` on [pattern-library](https://github.com/holidayextras/pattern-library) & pull this branch of brands.
- In the `pattern-library` directory, `npm link ../brands`
- Change `pattern-library/less/heha-theme:72` HEHA Theme section to be

```less
//
// HEHA Theme
// --------------------------------------------------

@import '../node_modules/brands/brands/heha/src/less/theme';

@asset-path: "https://d2r3bolva79zkl.cloudfront.net/assets/";
@asset-image-path: @asset-path;
@asset-font-path: "@{asset-path}fonts";
```

- `npm start` in pattern library & check out the planner. [http://localhost:3000/pages/planner/index.html](http://localhost:3000/pages/planner/index.html) Make sure you have switched to the HEHA! brand in the side menu.

#### Screenshots / Screencast
![planner](https://cldup.com/flVJ7gBMaV.png)

#### What gif best describes how you feel about this work?
![removal van](http://www.bestremovalservices.com/Images/van2.gif)

---


#### Reviewers

**Review 1**

**Review 2** \*

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
